src/LinuxTracing/LinuxTracingUtils.cpp

```diff
@@ -333,4 +334,36 @@ std::map<uint64_t, std::string> FindFunctionsThatUprobesCannotInstrumentWithMess
   return function_ids_to_error_messages;
 }
 
+absl::flat_hash_map<pid_t, pid_t> RetrieveInitialTidToRootNamespaceTidMapping(pid_t target_pid) {
+  absl::flat_hash_map<pid_t, pid_t> tid_mapping;
+  for (pid_t tid : orbit_base::GetTidsOfProcess(target_pid)) {
```

Comment:
And I would change this `tid` to `tid_in_root_namespace`.

Reply:
done